src/lib/profiles/data-management/Current/Command.cpp

```diff
@@ -37,13 +37,14 @@
 
 #include <SystemLayer/SystemStats.h>
 
-#if WEAVE_CONFIG_ENABLE_RELIABLE_MESSAGING && WDM_PUBLISHER_ENABLE_CUSTOM_COMMANDS
+#if WEAVE_CONFIG_ENABLE_RELIABLE_MESSAGING && WDM_PUBLISHER_ENABLE_CUSTOM_COMMAND_HANDLER
 
 namespace nl {
 namespace Weave {
 namespace Profiles {
 namespace WeaveMakeManagedNamespaceIdentifier(DataManagement, kWeaveManagedNamespaceDesignation_Current) {
 
+using namespace nl::Weave;
```

Comment:
From previous PR (@robszewczyk):

Given the editing changes elsewhere, please remove the using decaration

Reply:
Rob, there are many other cpp files with "using namespace" - why are you requesting that it be removed here?